Ext/Compose/README.md

```diff
@@ -55,7 +55,7 @@ class ComposableScreenshotTest {
 
     MIT License
     
-    Copyright (c) 2021 Shopify
+    Copyright (c) 2022 ndtp
```

Comment:
Copyright  on original works still must be attributed to Shopify prior to the fork. We'll need to represent this throughout with something like this.

```suggestion
    Copyright (c) 2022 ndtp
    Copyright (c) 2018-2022 Shopify
```

Reply:
I looked up a guide on copyright and it implied that, once you change a file, the old copyright is invalidated.
I don't mind keeping the Shopify copyright, but at what point is it a new file? Everything has been renamed, so it's technically not the same file anymore.
Do you have any references on how to handle this? I'll see if I can dig up the document I read.

@jaredh

Reply:
IANAL but the [MIT license](https://opensource.org/licenses/MIT) states

> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
> 
> **The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.**

I think headers are fine to scrap in general as long as attribution remains in `LICENSE`.

Reply:
None of this is very clear to me. I'll redo/change/update anything here that I need to. I don't want to get any lawyers after us, but man, it's confusing to me.

See, like I read that and think "substantial portions" means that, once you cross a threshold, it doesn't matter anymore ...
But it could also be a single function that remains unchanged. I'd rather be safe-than-sorry, but I also don't want to attribute ownership to Shopify if they shouldn't have it.

I'm thinking changing everything to ` Copyright (c) 2022 ndtp, Copyright (c) 2018-2022 Shopify` for any files that were migrated over from the old repo and use only `Copyright (c) 2022 ndtp` for any entirely new files.

@jaredh

Reply:
This [SE discussion](https://softwareengineering.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project) has a nice suggestion of differentiating with "Original work" and "Modified work".

> I'm thinking changing everything to Copyright (c) 2022 ndtp, Copyright (c) 2018-2022 Shopify for any files that were migrated over from the old repo and use only Copyright (c) 2022 ndtp for any entirely new files.

+1, I think that's the safest approach.

Reply:
Really like that SO suggestion. I think I'll do that. I can't see any problem with it

For any migrated/modified files, I'll use:

```
Original work Copyright (c) 2018 Shopify 
Modified work Copyright (c) 2022 ndtp
```

For new files, I'll use:
```
Copyright (c) 2022 ndtp
```

And I'll basically double the LICENSE.md file to include both an ndtp section and a Shopify section.

@jaredh

Reply:
I made the changes here https://github.com/ndtp/android-testify/pull/30

I'll re-do this PR once #30 is merged.